test/src/main/java/org/apache/accumulo/test/proxy/SimpleProxyBase.java

```diff
@@ -2077,7 +2077,8 @@ public void bulkImport() throws Exception {
 
     // Write an RFile
     String filename = dir + "/bulk/import/rfile.rf";
-    FileSKVWriter writer = FileOperations.getInstance().openWriter(filename, fs, fs.getConf(), null, DefaultConfiguration.getInstance());
+    FileSKVWriter writer = FileOperations.getInstance().openWriter().ofFile(filename, fs, fs.getConf())
+        .withTableConfiguration(DefaultConfiguration.getInstance()).execute();
```

Comment:
Within the Accumulo source, the only time that the "default" configuration is the right answer is in tests.  Having `FileAccessOperations` provide the default configuration might make writing some tests a tad easier, but at the cost of making it easier to write buggy functional code.